loggroup-lambda-connector/sam/packaged.yaml

```diff
@@ -39,7 +39,7 @@ Parameters:
     - Kinesis
   DestinationArnValue:
     Type: String
-    Default: arn:aws:lambda:us-east-1:123456789000:function:TestLambda
+    Default: arn:aws:lambda:us-west-2:123456789000:function:TestLambda
```

Comment:
This is not the KBS AWS account is it? I would expect the arn to start with `arn:aws:lambda:us-west-2:529513974030`

Reply:
This is overridden anyway, but I'll change the default to use our account.